stabilizer/src/main/java/com/hazelcast/stabilizer/worker/selector/OperationSelectorBuilder.java

```diff
@@ -0,0 +1,86 @@
+package com.hazelcast.stabilizer.worker.selector;
+
+import org.apache.log4j.Logger;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import static java.lang.String.format;
+
+/**
+ * Builder to create an {@link OperationSelector} instance. Each operations is represented as an enum item
+ * and is registered with its probability percentage (0 to 100) via the {@link #addOperation(Enum, int)} method.
+ * <p/>
+ *
+ * @param <T> enum of operations
+ */
+public class OperationSelectorBuilder<T extends Enum<T>> {
```

Comment:
Why isn't the `OperationSelector` thread-safe? It has just immutable fields and should be safely constructed. You may want to create multiple instances to reduce contention on `Random`, but it shouldn't be necessary.